packit/base_git.py

```diff
@@ -351,21 +351,40 @@ def fetch_upstream_archive(self):
         with cwd(self.absolute_specfile_dir):
             self.download_remote_sources()
 
-    def set_specfile_content(self, specfile: Specfile, version: str, comment: str):
+    def set_specfile_content(
+        self,
+        specfile: Specfile,
+        version: str,
+        comment: str,
+        preserve_autochangelog: bool = False,
+    ):
         """
```

Comment:
just a note that we use google-style docstrings and update old ones „on the go“

Reply:
Ok!